bin/wserver/wserver.mli

```diff
@@ -2,7 +2,7 @@
 
 (* module [Wserver]: elementary web service *)
 
-(** [Wserver.f syslog addr port tmout maxc g]
+(** [Wserver.create syslog addr port tmout maxc g]
```

Comment:
One argument is missing here.

Reply:
Yes Working_directory is missing from the comment. To add. ...

Reply:
Done last commit